static/scss/icomoon.scss

```diff
@@ -1,12 +1,6 @@
 // sass-lint:disable mixins-before-declarations
 @font-face {
   font-family: "icomoon";
-  src: url("#{$static-path}/fonts/icomoon.eot?dd3z7r"),
-  url("#{$static-path}/fonts/icomoon.eot?dd3z7r#iefix")
-  format("embedded-opentype"),
-  url("#{$static-path}/fonts/icomoon.ttf?dd3z7r") format("truetype"),
-  url("#{$static-path}/fonts/icomoon.woff?dd3z7r") format("woff"),
-  url("#{$static-path}/fonts/icomoon.svg?dd3z7r#icomoon") format("svg");
```

Comment:
This might not fix everything we want to. Refer to the lighthouse report which still flags this as an issue:

<img width="712" alt="Screenshot 2020-09-24 at 5 20 35 PM" src="https://user-images.githubusercontent.com/6387678/94144191-441e4d80-fe8a-11ea-8a53-f4ba7e2a7eba.png">

The recommended way of fixing it:
https://web.dev/font-display/?utm_source=lighthouse&utm_medium=devtools

Reply:
It's done exactly that way and I don't see that error on my side.